include/uv.h

```diff
@@ -1096,6 +1104,9 @@ struct uv_fs_s {
   void* ptr;
   const char* path;
   uv_stat_t statbuf;  /* Stores the result of uv_fs_stat() and uv_fs_fstat(). */
+  const uv_dir_t* dir; /* Stores the result of uv_fs_opendir() */
+  uv_dirent_t* dirents;
+  size_t nentries;
```

Comment:
Yup, I didn't see this.  _(I actually overlooked it twice because of the styling of the GitHub diff viewer...and this PR is pretty old so I just forgot.)_